### PR TITLE
Adds Docker section

### PR DESCRIPTION
**Pull request by ravikiran-sulikeri**

### CODE REVIEW

File: modules/ROOT/pages/appendix/faq.adoc [hunk 1]
```diff
@@ -8,8 +8,8 @@
 ** No, currently NOM does not officially support desktop at this time.
 
 * Can I deploy NOM in a cloud environment?
-** It is possible to deploy NOM into a supported cloud environment, using tarball and zip installations.
-Docker and Kubernetes support will be added later.
+** It is possible to deploy NOM into a supported cloud environment, using tarball and zip installations. It is also possible to deploy NOM server as a docker container if the cloud environment uses containers.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/appendix/faq.adoc:11]:
```suggestion
** It is possible to deploy NOM into a supported cloud environment, using tarball and zip installations. 
It is also possible to deploy a NOM server as a docker container if the cloud environment uses containers.
```
One sentence per line.

File: modules/ROOT/pages/docker/server.adoc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+= Server
+:description: This page describes how to run NOM server in a container.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/docker/server.adoc:2]:
```suggestion
:description: This page describes how to run a NOM server in a container.
```

File: modules/ROOT/pages/introduction/deployment.adoc [hunk 3]
```diff
@@ -7,7 +7,10 @@ These may be deployed in a number of ways depending on the platform and technolo
 Initially, NOM will be made available for customers deploying in self-managed environments, including in their own data centers (on-premises), private cloud or on one of the Neo4j supported public clouds.
 
 Currently, support is available for Linux and Windows using tarball or zip file installations.
-Support for package managers will be added in a later release, along with support for docker and kubernetes.
+
+The NOM server could be deployed as a docker container with newly published images.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/introduction/deployment.adoc:11]:
```suggestion
The NOM server can be deployed as a Docker container with newly published images.
```

File: modules/ROOT/pages/docker/index.adoc [hunk 4]
```diff
@@ -0,0 +1,6 @@
+= Docker
+:description: This chapter describes the NOM containerization.
+
+This chapter describes the following::
+* xref:./introduction.adoc[Introduction] - Dockerizing the server
+* xref:./server.adoc[Server] - Running a NOM server in a docker container
```
Comment (AlexicaWright) [inline modules/ROOT/pages/docker/index.adoc:6]:
```suggestion
* xref:./server.adoc[Server] - Running a NOM server in a Docker container
```

File: modules/ROOT/pages/introduction/deployment.adoc [hunk 5]
```diff
@@ -7,7 +7,10 @@ These may be deployed in a number of ways depending on the platform and technolo
...
+
+Support for package managers will be added in a later release, along with support for kubernetes.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/introduction/deployment.adoc:13]:
```suggestion
Support for package managers will be added in a later release, along with support for Kubernetes.
```

**Review by AlexicaWright — COMMENTED** [review]
Some editorial and grammatical comments.

**Review by eve-bright — COMMENTED** [review]
I am not sure about whether Docker should be it's own section or if it should be a sub-section of Installation? @AlexicaWright do you have any thoughts on that?

**Comment by davidoliverSP2** [comment]
> I am not sure about whether Docker should be it's own section or if it should be a sub-section of Installation? @AlexicaWright do you have any thoughts on that?

In the main Neo4j docs, we decided that it should be a peer to Installation, not a sub-section. The reasoning was that using Docker images is an alternative to installation, not part of installation. We did add a note to the main installation page though that directed users who didn't want to manage their own installations to either the Docker instructions, or to Aura: https://neo4j.com/docs/operations-manual/current/installation/

**Comment by AlexicaWright** [comment]
In the Operations manual, the Docker section is separate from Installation section and in the Bloom docs, it is part of Installation. I suppose it comes down to the length of the Docker section. In this case, as the section is larger than the Installation section, I feel like it would be acceptable to keep it separate.

**Review by eve-bright — APPROVED** [review]

**Pull request merged**